packages/firestore/src/typings/firestore.d.ts

```diff
@@ -544,9 +544,13 @@ declare namespace firestore {
    * A `DocumentSnapshot` contains data read from a document in your Firestore
    * database. The data can be extracted with `.data()` or `.get(<field>)` to
    * get a specific field.
+   *
```

Comment:
So in firestore-api-changes, I believe our .d.ts types are repeated in 3 places for good measure (packages/firebase/app/index.d.ts, packages/firebase/index.d.ts, packages/firestore/src/typings/firestore.d.ts), but Josh just merged his typings refactor into master which unifies them to a single place (packages/firestore-types/index.d.ts).  So we probably should merge master into firsetore-api-changes and then rework your change on that.

Reply:
I merged Josh's PR and changes this in just one place. Note that we will have to update firestore-externs.js when we decide to merge this branch into master.